addons/attributes/script_component.hpp

```diff
@@ -2,9 +2,9 @@
 #define COMPONENT_BEAUTIFIED Attributes
 #include "\x\zen\addons\main\script_mod.hpp"
 
-// #define DEBUG_MODE_FULL
-// #define DISABLE_COMPILE_CACHE
-// #define ENABLE_PERFORMANCE_COUNTERS
+#define DEBUG_MODE_FULL
+#define DISABLE_COMPILE_CACHE
+#define ENABLE_PERFORMANCE_COUNTERS
```

Comment:
Should be disabled.

Reply:
Not finished yet. Only ready for feedback on general functionality, code still needs some work/cleanup.

Reply:
Yeah, if it was a draft PR then you could leave it enabled but the it's no longer a draft.